cmd/sshpiperd/daemon.go

```diff
@@ -217,19 +218,32 @@ func (d *daemon) run() error {
 			var uphook func([]byte) ([]byte, error)
 			var downhook func([]byte) ([]byte, error)
 
-			if d.recorddir != "" {
-				recorddir := path.Join(d.recorddir, p.DownstreamConnMeta().User())
+			uniqID := plugin.GetUniqueID(p.ChallengeContext())
```

Comment:
hi 
typically, the id is a guid, any idea how to manage those folders? any meta/index info?

Reply:
I think the metadata should be managed by the plugin, for example, writing the log path and the user_id into the database. Alternatively, we could also include the SSH username in the file path for easier management?